detect_variants/detect_variants.cwl

```diff
@@ -32,6 +32,24 @@ inputs:
         type: int?
     mutect_artifact_detection_mode:
         type: boolean?
+    mutect_max_alt_allele_in_normal_fraction:
+        type: float?
+    mutect_max_alt_alleles_in_normal_count:
+        type: int?
+    varscan_strand_filter:
+        type: int?
+        default: 0
```

Comment:
hmm, to my understanding, the upper layer CWLs: detect_variants.cwl, varscan/workflow.cwl, varscan.cwl should work without the defaults set since the bottom layer CWL varscan_somatic.cwl has the defaults set. These parameter inputs need to be optional across the layers. I will make the change and test it if this is correct.